SConstruct

```diff
@@ -515,6 +486,37 @@ if selected_platform in platform_list:
         # We apply it to CCFLAGS (both C and C++ code) in case it impacts C features.
         env.Prepend(CCFLAGS=["/std:c++17"])
 
+    # 'dev' and 'production' are aliases to set default options if they haven't been set
+    # manually by the user.
+    if env["dev"]:
+        env["verbose"] = methods.get_cmdline_bool("verbose", True)
+        env["warnings"] = ARGUMENTS.get("warnings", "extra")
+        env["werror"] = methods.get_cmdline_bool("werror", True)
+        if env["tools"]:
+            env["tests"] = methods.get_cmdline_bool("tests", True)
+    if env["production"]:
+        env["use_static_cpp"] = methods.get_cmdline_bool("use_static_cpp", True)
+        env["lto"] = ARGUMENTS.get("lto", "thin" if (methods.using_clang(env) or methods.using_emcc(env)) else "full")
```

Comment:
Had to move the block after parsing the platform `configure` so that `methods.using_clang(env) or methods.using_emcc(env)` works (on Linux/Windows it depends on handling `use_llvm=yes` in `configure`).

As a side note, we really need better code to handle knowing which toolchain we're using.